hack/minikube-rook-mirror-test.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -x
+set -e -o pipefail
+
+## Variables
+PRIMARY_CLUSTER="${PRIMARY_CLUSTER:-hub}"
+SECONDARY_CLUSTER="${SECONDARY_CLUSTER:-cluster1}"
+STORAGECLASS_NAME="rook-ceph-block"
+PVC_NAME="rbd-pvc"
+
+## Usage
+usage()
+{
+	set +x
+	echo "Usage:"
+	echo "  $0"
+	echo "  Available environment variables:"
+	echo "    minikube primary cluster PRIMARY_CLUSTER ${PRIMARY_CLUSTER}"
+	echo "    minikube secondary cluster SECONDARY_CLUSTER ${SECONDARY_CLUSTER}"
+	exit 0
+}
+
+cat <<EOF | kubectl --context="${PRIMARY_CLUSTER}" apply -f -
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: "${PVC_NAME}"
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: "${STORAGECLASS_NAME}"
+EOF
+
+cat <<EOF | kubectl --context="${PRIMARY_CLUSTER}" apply -f -
+apiVersion: replication.storage.openshift.io/v1alpha1
+kind: VolumeReplication
+metadata:
+  name: vr-sample
+spec:
+  volumeReplicationClass: vrc-sample
+  replicationState: Primary
+  dataSource:
+    kind: PersistentVolumeClaim
```

Comment:
need to add API group or else it will fail https://github.com/csi-addons/volume-replication-operator/issues/56

Reply:
🤔 This passed without it for me. The spec [here](https://github.com/kubernetes/kubernetes/blob/01289178ea58df0520ec9cea0ce4bdc8aa6e93f1/staging/src/k8s.io/api/core/v1/types.go#L5417-L5421) states it is optional but there is no json `omitempty` here.


Is it a case of kubernetes versions?

Reply:
Yes, it was failing earlier, even I don't see this error. it might be fixed in the recent version